pkg/virt-config/config-map.go

```diff
@@ -461,6 +463,53 @@ func setConfigFromKubeVirt(config *v1.KubeVirtConfiguration, kv *v1.KubeVirt) er
 	return nil
 }
 
+// setConfigFromCPUArch setup arch specific config and updates the provided config.
+func setConfigFromCPUArch(config *v1.KubeVirtConfiguration) error {
```

Comment:
Consider naming this `setCPUArchSpecificDefault`

Reply:
Consider giving `defaultClusterConfig` an arch argument. Then the whole `defaulting` can be done like before, and we only override with user-overrides then.